go/oasis-node/cmd/node/node.go

```diff
@@ -359,7 +369,9 @@ func (n *Node) initGenesis() error {
 //
 // WARNING: This will misbehave iff cmd != RootCommand().  This is exposed
 // for the benefit of tests and the interface is not guaranteed to be stable.
-func NewNode() (*Node, error) {
+//
+// testNode is true when the node is used in unit tests.
+func NewNode(testNode bool) (*Node, error) {
```

Comment:
I think it would be nicer to have these as:
* `newNode(testNode bool)` (internal, not exported)
* `NewNode()`
* `NewTestNode()`